src/config/project/mod.rs

```diff
@@ -15,6 +15,7 @@ pub struct ProjectConfig {
     pub repository: Option<String>,
     pub version: Option<String>,
     pub license: Option<String>,
+    pub readme_path: Option<Utf8PathBuf>,
```

Comment:
can you say more about adding this here? does this connect to axoproject? (it's not clear to me we ever receive this value when deserializing a project manifest so i want to make sure it's connected)

Reply:
It's a value that Cargo.toml lets you specify and axoproject already reads. I did a quick pass of "are there values we're not exposing that we already have equivalents of in oranda" and this is the only one that turned up.